internal/image/validate.go

```diff
@@ -36,14 +37,33 @@ func ValidateImage(ctx context.Context, imageRef, policyConfiguration, publicKey
 		return nil, err
 	}
 
+	if err = a.ValidateImageAccess(ctx); err != nil {
+		log.Debugf("Image access check failed. Error: %s", err.Error())
+		out.SetImageAccessibleCheck(false, fmt.Sprintf("image ref not accessible. %s", err))
+		out.SetImageSignatureCheck(false, "skipped due to inaccessible image ref")
```

Comment:
Could it be that adding the "skipped" messages under violations may cause confusion?

Reply:
Would you rather them be left empty?

Reply:
Not sure. I think arguably, "skipped" is not really a violation, but I see the motivation for adding it here. Maybe a debug message is a better approach? Either way is fine. We can always tweak later after some experience over time.

Reply:
I've left them empty for now.